hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/TestFsDatasetImpl.java

```diff
@@ -1173,7 +1173,7 @@ private void testMoveBlockFailure(Configuration config) {
           .getReplicaInfo(block.getBlockPoolId(), newReplicaInfo.getBlockId())
           .getGenerationStamp());
       LambdaTestUtils.intercept(IOException.class, "Generation Stamp "
-              + "should be monotonically increased.",
+              + "should be monotonically increased",
```

Comment:
backmerge needed ?

Reply:
Yes, the base earlier was different, hence this added codepiece. Have rebased accordingly now, it has been removed.